bin/cron/update_project_count

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+#
+#
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/only_one'
+require 'cdo/properties'
+require src_dir 'database'
+
+PEGASUS_DB = Sequel.connect(CDO.pegasus_db_writer.sub('mysql:', 'mysql2:'))
+STORAGE_APPS = PEGASUS_DB[:storage_apps]
+
+# This default number of lines of code is from 2019-1-21.
+DEFAULT_NUMBER_OF_PROJECTS = 35_000_000
+
+def main
+  project_count = 0
+  # Get the current amount
+  STORAGE_APPS.each do |row|
```

Comment:
i _think_ we can simplify this query to a bulk query rather than iterating over each row in the storage_apps table. @sureshc thoughts/ideas on the best way to count the number of standalone projects in the storage_apps table using Sequel?

Reply:
it looks like `STORAGE_APPS.where(standalone: true).count` would work, although i'm not very familiar with Sequel

Reply:
Good call.  It's possible to turn on logging for the database and then view the SQL generated.  We should definitely make sure that we are generating a query that does the counting in SQL using a simple query rather than doing a ton of incrementing in Ruby.

Reply:
Definitely open to doing this a different way - this was the general way poorva had suggested to find it.  Should I switch it over to your suggestion @madelynkasula ?

Reply:
i'd like to hear suresh's thoughts/concerns with the approach i mentioned first. it may be doable (mostly because this is a job that only runs at 4:00 UTC on Sundays), or we might have to rethink this query if performance is still a concern

Reply:
Given the open question, I think we should examine the generated SQL on localhost regardless.  It looks pretty likely that we're going to have a 35 million+ row object and we're going to be iterating through it that many times :)  This is definitely something that SQL should be able to take care of for us.  

I'm also not sure the exact Sequel query, but we should be able to play around in `pegasus-console` quite easily, while examining the logged SQL queries at the same time, to figure out a performant optimal query here.  

btw, I think you can turn on logging for your local `pegasus-sql` with `SET general_log = 1;` and then tail the log.  On my machine, that is done with `tail -f /usr/local/var/mysql/my-machine-name.log`.

Reply:
I think we can do something like:
`project_count = PEGASUS_REPORTING_DB_READER[:storage_apps].where(standalone: true).count`

The Sequel documentation [has good examples](url
https://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-count) of how to carry out common types of queries.